src/customerio-cdp.ts

```diff
@@ -31,7 +31,8 @@ export class CustomerIO {
     // if (config.logLevel && config.logLevel !== CioLogLevel.None) {
     //   NativeLoggerListener.initialize();
     // }
-    NativeCustomerIO.initialize(config, config.logLevel);
+    let logLevel = config.logLevel?.valueOf() ?? CioLogLevel.None.valueOf();
```

Comment:
think our default is `ERROR`

Reply:
Agreed. Our default is `Error`across platforms. However, I was thinking from a customer's POV that if they do not provide a logLevel then I expect that they do not need the logs hence `None` but I have no strong opinion in keeping it as `Error`. Updated! ✅